### PR TITLE
Refactor TestUtils to work with old tests and new incoming ones

### PR DESCRIPTION
**Pull request by emilyvera**

### CODE REVIEW

File: src/test/java/TestUtils.java [hunk 1]
```diff
@@ -29,27 +29,47 @@ private TestUtils() {
...
+            createEntity(/* userId = */ "123", /* timestamp = */ 1491582960000L,
+                new BlobKey("test"), "img/restaurant-receipt.jpeg", 29.01, "main street restaurant",
+                ImmutableSet.of("food", "meal", "lunch"), ""));
+
+    return entities;
+  }
+
+  /** Creates and returns a single Receipt entity. */
+  public static Entity createEntity(String userId, long timestamp, BlobKey blobkey, String imageUrl,
+      Double price, String store, ImmutableSet<String> categories, String rawText) {
```
Comment (basaam0) [inline src/test/java/TestUtils.java:72]: Is there a reason for using the `Double` wrapper class instead of `double` here?

Comment (avagarcia0) [reply src/test/java/TestUtils.java:72]: It looks like a typo given that the createEntity calls all use a `double`.

Comment (emilyvera) [reply src/test/java/TestUtils.java:72]: In #87 there's a test where price is null. The Double wrapper here will allow for this null check

File: src/test/java/TestUtils.java [hunk 2]
```diff
@@ -29,27 +29,47 @@ private TestUtils() {
...
+
+            createEntity(/* userId = */ "123", /* timestamp = */ 1491582960000L,
+                new BlobKey("test"), "img/restaurant-receipt.jpeg", 29.01, "main street restaurant",
+                ImmutableSet.of("food", "meal", "lunch"), ""));
+
+    return entities;
+  }
+
+  /** Creates and returns a single Receipt entity. */
+  public static Entity createEntity(String userId, long timestamp, BlobKey blobkey, String imageUrl,
```
Comment (acheng1) [inline src/test/java/TestUtils.java:71]: See my comments in https://github.com/googleinterns/step12-2020/pull/87 ...

**Pull request merged**